src/renderer.ts

```diff
@@ -272,7 +272,8 @@ class Renderer extends EventEmitter<RendererEvents> {
   }
 
   getWidth(): number {
-    return this.scrollContainer.clientWidth
+    const { paddingLeft, paddingRight } = getComputedStyle(this.scrollContainer)
+    return this.scrollContainer.clientWidth - parseFloat(paddingLeft) - parseFloat(paddingRight)
```

Comment:
Hi @JanMisker, sorry again for the huge delay.
This function is called on each scroll in Regions and Timeline so we have to be careful with performance implications here. Since it's not reading the DOM element's dimensions directly like clientWidth does, it would make sense to cache these computed styles. They aren very unlikely to change during a wavesurfer instance lifecycle, so they can be cached just once and never invalidated.